packages/TelegramClient-Core.package/TCCMessage.class/README.md

```diff
@@ -1 +1 @@
-super-dataclass for messages of all types
\ No newline at end of file
+class with a very abstract of a message
```

Comment:
We should make this a bit clearer, maybe "Event that occurs in a chat at a specific time"?

Reply:
Should be clearer now and consistent with the comments of the sub classes